apps/heatmap/init.js

```diff
@@ -25,6 +25,10 @@ function initialize() {
       initUIcomponents();
       // create a viewer and set up
       initCore();
+      setTimeout(() => {
+        // initialize sockets
```

Comment:
@Vedant1202  Just curious what's the reason for forcing this delay?

Reply:
Oh that's still work in progress.. I suppose some data that I need isn't showing up at the correct time